src/styles/FontUploader.css

```diff
@@ -2,24 +2,29 @@
     display: flex;
     flex-direction: column;
     align-items: center;
-    margin: 20px;
+    margin: 1.25em;
 }
 
 .drop-area {
     display: flex;
     flex-direction: column;
     align-items: center;
-    border: 2px dashed #ddd;
-    padding: 20px;
+    border: 2px dashed #808080;
+    background-color: white;
+    border-radius: 10px;
     text-align: center;
     cursor: pointer;
+    height: 14vh;
+    width: 60vh;
 }
 
-.selected-fonts {
-    margin-top: 10px;
+.input-button {
+    padding: 0.75em;
+    margin-block-start: 0.75em;
+    padding-inline-start: 6.5em;
 }
 
-.input-button {
-    margin-block-start: 10px;
-    padding-inline-start: 40px;
+.descriptor {
+    margin-block-start: 1em;
+    font-size: large;
```

Comment:
`large` can vary browser-to-browser. A `rem` value might help with layout predictability here.